libcontainer/system/linux.go

```diff
@@ -9,13 +9,29 @@ import (
 	"os"
 	"os/exec"
 	"strconv"
+	"sync/atomic"
 	"syscall"
 	"unsafe"
 
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
 )
 
+//go:linkname syscallOrigRlimitNofile syscall.origRlimitNofile
```

Comment:
While not required, it's better for readability to put go:linkname line just before the variable declaration.

Reply:
Because there should be no space between double slash and `go:linkname`. This looks ugly with other comments before the variable declaration.

Reply:
Thanks.